{"name": "Проверка создания заказа неавторизованного пользователя c неверным хешем ингредиентов", "status": "failed", "statusDetails": {"message": "AssertionError: assert (400 == 500)\n +  where 400 = <Response [400]>.status_code", "trace": "self = <test_order.TestCreatingOrder object at 0x000001F95BADB3B0>\n\n    @allure.title('Проверка создания заказа неавторизованного пользователя c неверным хешем ингредиентов')\n    @allure.description('Нельзя создать заказ неавторизованным пользователем c неверным хешем ингредиентов')\n    def test_creating_order_not_auth_user_invalid_hash_error_message(self):\n        payload = {\"ingredients\": [\"123456a71d1f82001bdaaa6d\"]}\n        response_post = requests.post(Constants.MAIN_URL + APIOrderUrls.CREATING_ORDER_URL, data=payload)\n>       assert (response_post.status_code == 500 and response_post.json()[\"success\"] == False\n                and response_post.json()[\"message\"] == APIOrderResponseTexts.CREATING_ORDER_INVALID_HASH_ERROR)\nE       assert (400 == 500)\nE        +  where 400 = <Response [400]>.status_code\n\ntests\\test_order.py:68: AssertionError"}, "description": "Нельзя создать заказ неавторизованным пользователем c неверным хешем ингредиентов", "start": 1723215304589, "stop": 1723215304793, "uuid": "57ba0ba2-4a26-4745-ba7f-2a53dbacb4f0", "historyId": "f688655f0362443cdbcfda16f94490a4", "testCaseId": "f688655f0362443cdbcfda16f94490a4", "fullName": "tests.test_order.TestCreatingOrder#test_creating_order_not_auth_user_invalid_hash_error_message", "labels": [{"name": "epic", "value": "test_creating_order"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestCreatingOrder"}, {"name": "host", "value": "DESKTOP-619AU3M"}, {"name": "thread", "value": "12804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order"}]}